### DISCUSSION TITLE
Shared link only shows relational and dropdown fields

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13923] [Archive]
With a shared link created in a collection (with no pass word), when out of Directus, I only see the relational fields and the dropdows.
All other inputs, textareas are blank. The role attached to the link has `Read` set to `All access`
I thought that the shared link would allow anyone, even not registered.

**Comment by rijkvanzanten** [comment]
Mind opening a bug report? Sounds like something isn't working as expected

**Reply by Pachat** [reply]
Done  #13971
